exatomic/exa/__init__.py

```diff
@@ -14,8 +14,7 @@
 with open(os.path.join(os.path.dirname(__file__),
           'conf', 'logging.yml'), 'r') as f:
     _log = yaml.safe_load(f.read())
-_log['handlers']['file']['filename'] = os.path.join(tempfile.gettempdir(), 'exa.log')
+_log['handlers']['file']['filename'] = os.path.join(tempfile.gettempdir(), 'exatomic.exa.log')
```

Comment:
logging needs some love anyway but probably can flatten to exatomic.log

Reply:
left as `exa.log` to avoid collision - will open issue to refactor.